applications/connecting_applications_to_services/getting-started-with-service-binding.adoc

```diff
@@ -17,7 +17,7 @@ The {servicebinding-title} manages the data plane for workloads and backing serv
 * You have access to an {product-title} cluster using an account with `cluster-admin` permissions.
 * You have installed the `oc` CLI.
 * You have installed {servicebinding-title} from OperatorHub.
-* You have installed the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* Update channel. The installed Operator is available in an appropriate namespace, such as the `my-petclinic` namespace.
+* You have installed the 5.1.2 version of the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* Update channel. The installed Operator is available in an appropriate namespace, such as the `my-petclinic` namespace.
```

Comment:
Do we have to be specific about the version, is there a range we can specify?

Reply:
@yes, it is the minimum expected, and hence want to mention it explicitly after discussion with @pmacik and testing with the specific images for 4.11.

Reply:
If we do not pin the version of the operator, the OLM will install the latest version of that operator. Then the follwing instructions (specifically the step to install a CR of the DB instance) will become incompatible with the installed version of the operator and the quickstart would stop working.

Reply:
Can we say in the doc that this is a min version: install version 5.1.2 or later

Reply:
@pmacik Please respond to [this query.](https://github.com/openshift/openshift-docs/pull/50500#discussion_r974310165)

Reply:
@dperaza4dustbit that would cause the same problems as the "latest". Only when a concrete version (or versions) are specified you can guarantee that the procedure works with the data (CR) provided.

Reply:
OK